kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/EnvelopeEncryptionST.java

```diff
@@ -54,6 +56,10 @@ void setUp() {
         else {
             NamespaceUtils.createNamespaceWithWait(Vault.VAULT_DEFAULT_NAMESPACE);
             kubeVaultTestKmsFacade.start();
+            assumeThat(kubeVaultTestKmsFacade.isCorrectVersionInstalled())
```

Comment:
Why assumeThat?  Surely we want to fail if there's a conflict.  assumeThat skips the tests.